karton/classifier/classifier.py

```diff
@@ -397,6 +397,7 @@ def zip_has_mac_app() -> bool:
             "xz": ["XZ compressed data"],
             "zip": ["Zip archive data"],
             "zlib": ["zlib compressed data"],
+            "lzh": ["  LHa (2.x) archive data"],
```

Comment:
```suggestion
            "lzh": ["LHa (2.x) archive data"],
```

Is this space here on purpose?

Reply:
Hm, It seems that it is 🤔
Filemagic is weird